### PR TITLE
chore: enable Git LFS for WireReusableUIComponents' snapshot reference images - WPB-10959

### PR DESCRIPTION
**Pull request by caldrian**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-10959" title="WPB-10959" target="_blank">TaskWPB-10959</a>  Enable Git LFS
  </td></table>
  <br />
 

### Issue

The Git repository has grown unnecessarily much by adding and updating snapshot reference images.
This PR enables Git LFS for a subset of images.

Make sure to have `git-lfs` installed, e.g. by using Homebrew.
See the instructions on https://git-lfs.com.
```
brew install git-lfs
```
Afterwards run ./setup.sh again.

---

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [x] Adds/updates automated tests.

---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
- [x] Make sure you use the API for UI elements that support large fonts.
- [x] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
- [x] New UI elements have Accessibility strings for VoiceOver.

### CODE REVIEW

**Review by agisilaos — APPROVED** [review]

File: WireUI/Tests/WireReusableUIComponentsTests/Resources/ReferenceImages/AccountImageViewSnapshotTests/personal-available.dark.png
Comment (samwyndham) [inline WireUI/Tests/WireReusableUIComponentsTests/Resources/ReferenceImages/AccountImageViewSnapshotTests/personal-available.dark.png:1]: @caldrian Any idea what is happening here and why GitHub can't render?

Comment (samwyndham) [reply WireUI/Tests/WireReusableUIComponentsTests/Resources/ReferenceImages/AccountImageViewSnapshotTests/personal-available.dark.png:1]: Just wondering is there is a misconfiguration. When I've worked with LFS in the past GitHub has rendered images OK

Comment (caldrian) [reply WireUI/Tests/WireReusableUIComponentsTests/Resources/ReferenceImages/AccountImageViewSnapshotTests/personal-available.dark.png:1]: I experienced that whenever you change from LFS to non-LFS or vice versa, the rendering fails.

**Review by samwyndham — APPROVED** [review]
Great to be getting Git LFS!

File: setup.sh [hunk 1]
```diff
@@ -47,6 +47,14 @@ die "Xcode version for the repository should be at least ${repository_xcode_vers
 
 REPO_ROOT=$(git rev-parse --show-toplevel)
 
+if [[ -n "${CI-}" ]]; then
+    echo "Running on CI, skipping git lfs install"
+else
+    echo "ℹ️  Running git lfs install..."
+    git lfs install
```
Comment (netbe) [inline setup.sh:54]: is there a way to check git-lfs is installed?

right now I am getting :
```
› ./setup.sh                                                                                                                                                                                                                         08:50:37
ℹ️  Running git lfs install...
WARNING: You called a Git command named 'lfs', which does not exist.
Continuing in 0.1 seconds, assuming that you meant 'log'.
fatal: ambiguous argument 'install': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

also we can add it to readme :
`brew install git-lfs`

Comment (caldrian) [reply setup.sh:54]: > check for git-lfs

done: 8007fc8a0f1d6e6fcbaa63c1b7bb196388ea21aa

> update readme

done: 6e6496ba047aba50407b4992f8e0a9ef57be4f49

**Pull request merged**